cypress.config.js

```diff
@@ -14,9 +13,10 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
```

Comment:
The calculation of `randomNumber` is incorrect. The `Math.random(1000)` function does not take any arguments. You should use `Math.random()` and multiply the result by 1000, then use `Math.ceil()` to get a number between 1 and 1000.